containerfiles/initrd-builder/osbuilder/systemd/system/nvidia-cdi.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Generate NVIDIA CDI Configuration
+Wants=multi-user.target
```

Comment:
Is the dependency really on multi-user.target, or is it some specific NVIDIA systemd entry?